CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Added
 Changed
 =======
 - Limited maximum log message size to syslog hander in ``logging.ini`` template to 16000 characters. Syslog handler has potential to cause crashes with very long log messages, so operators should consider configuring the syslog message handler in a similar fashion.
+- Changed the default vlan tag range from [[1,4095]] to [[1, 4094]]
```

Comment:
Great compromise and solution @Ktmi. I'll leave it pre-approved. Before shipping this PR though:

- Let's also include a `topology` console DB script and then also link it here in the changelog. 

The script should detect if vlan 4095 has been allocated on any topology interface. It it's been, then let's check the flow owner too and list/print that information, and print a message asking network operators to first undo/remove/redeploy just so 4095 is no longer used. We'll recommend network operators to run this console script before upgrading to 2025.1, otherwise without this if vlan 4095 is used and later on released and tag ranges cleared on an interface it can lead to the issue we discussed.

Reply:
Script has been merged.